tests_e2e/tests/lib/agent_log.py

```diff
@@ -209,6 +209,15 @@ def get_errors(self) -> List[AgentLogRecord]:
                 'if': lambda r: DISTRO_NAME == 'ubuntu' and DISTRO_VERSION >= '22.00'
             },
             #
+            # Old daemons can produce this message
```

Comment:
Since the new test suite disables the agent, agent update does not happen and this message shows up in some distros.